x-pack/legacy/plugins/graph/public/app.js

```diff
@@ -630,33 +636,9 @@ app.controller('graphuiPlugin', function (
     $scope.detail = { mergeCandidates };
   };
 
-  // Zoom functions for the SVG-based graph
-  const redraw = function () {
-    d3.select('#svgRootGroup')
```

Comment:
Since you removed d3 completely from this file you could also remove the d3 import at line 8